website/users/models.py

```diff
@@ -44,11 +43,9 @@ def create_superuser(self, username, **kwargs):
         return self._create_user(username, **kwargs)
 
 
-class User(AbstractUser):
+class User(BaseUser):
```

Comment:
Are we actually using this class anymore? Because the settings file does not state a custom User model anymore.